packages/nimble-components/package.json

```diff
@@ -114,7 +114,7 @@
     "@storybook/addon-links": "^7.4.0",
```

Comment:
I'm attempting to fix the build issue in #1797

Reply:
That fixed the Angular build issue but the next build that ran seems to be hitting a Storybook breaking change:

 Error: src/banner/tests/banner.stories.ts(51,18): error TS2322: Type '<T extends Renderer>(storyFn: PartialStoryFn<T, Args>) => T["storyResult"]' is not assignable to type 'DecoratorFunction<HtmlRenderer, { open: boolean; title: string; text: string; severity: "error" | "default" | "warning" | "information"; action: ActionType; preventDismiss: boolean; titleHidden: boolean; toggle: unknown; usedLabels: null; }>'.
  Type 'unknown' is not assignable to type 'StoryFnHtmlReturnType'.

There are also Angular and Blazor build errors but they might be caused by the nimble-components build having failed.

I'm not going to start looking at it now but will add it to my backlog. If anyone wants to look at it sooner, feel free.

Reply:
There's also some weirdness where the PR description claims it's updating lots of storybook deps but the package file only shows a change to csf.

Reply:
I verified that the build issue still occurs with a manual update: #1803 

Looks like there is already a Storybook issue tracking this but it hasn't gotten much attention https://github.com/storybookjs/storybook/issues/25170

I'm following the issue so I can revive this PR if it gets fixed. Debugging this isn't high on my list for now so I'd propose we move this PR back to Draft.

Reply:
Seems to be building!